relayer/relays/beefy/polkadot-listener.go

```diff
@@ -162,8 +123,22 @@ func (li *PolkadotListener) queryBeefyAuthorities(blockHash types.Hash) ([]subst
 	return authorities, nil
 }
 
-func (li *PolkadotListener) queryBeefyNextAuthoritySet(blockHash types.Hash) (types.BeefyNextAuthoritySet, error) {
-	var nextAuthoritySet types.BeefyNextAuthoritySet
+func (li *PolkadotListener) queryBeefyAuthoritySet(blockHash types.Hash) (BeefyAuthoritySet, error) {
+	var authoritySet BeefyAuthoritySet
+	storageKey, err := types.CreateStorageKey(li.conn.Metadata(), "MmrLeaf", "BeefyAuthorities", nil, nil)
+	ok, err := li.conn.API().RPC.State.GetStorage(storageKey, &authoritySet, blockHash)
+	if err != nil {
+		return authoritySet, err
+	}
+	if !ok {
+		return authoritySet, fmt.Errorf("beefy authoritySet not found")
+	}
+
+	return authoritySet, nil
+}
+
+func (li *PolkadotListener) queryBeefyNextAuthoritySet(blockHash types.Hash) (BeefyAuthoritySet, error) {
+	var nextAuthoritySet BeefyAuthoritySet
 	storageKey, err := types.CreateStorageKey(li.conn.Metadata(), "MmrLeaf", "BeefyNextAuthorities", nil, nil)
```

Comment:
Seems `BeefyMmrLeaf` is the pallet prefix for polkadot network.

Reply:
This function is actually unused now.